src/components/common/YourCompany/CorrectName/CorrectName.vue

```diff
@@ -101,7 +101,7 @@ import CorrectNameToNumber from './CorrectNameToNumber.vue'
 })
 export default class CorrectName extends Vue {
   /** The options to display */
-  @Prop() readonly correctionNameChoices!: Array<string>
```

Comment:
This has nothing to do with "corrections", but I kept the prop as "correct name choices" because that's the name of the component.